draft-ietf-tls-esni.md

```diff
@@ -420,6 +420,19 @@ retry_configs
 decreasing order of preference, to be used by the client in subsequent
 connection attempts. These are known as the server's "retry configurations".
 
+Finally, when the client offers the "encrypted_client_hello", if the payload is
+the `inner` variant and the server responds with HelloRetryReqwuest, it MUST
+include an "encrypted_client_hello" with the following payload:
+
+~~~
+    struct {
+       opaque confirmation[8];
+    } ECHHelloRetryRequest;
```

Comment:
Why isn't this in a `select {}` block for the extension?

Reply:
[We had a thread on this, but I accidentally resolve it and I can no longer find it. So ist das Leben.] We're following the precedent set by the "key_share" extension https://datatracker.ietf.org/doc/html/rfc8446#section-4.2.8.